edit-post/components/sidebar/post-schedule/index.js

```diff
@@ -14,16 +14,20 @@ export function PostSchedule() {
 	return (
 		<PostScheduleCheck>
 			<PanelRow className="edit-post-post-schedule">
-				<span>{ __( 'Publish' ) }</span>
+				<span id="edit-post-post-schedule__heading">{ __( 'Publish' ) }</span>
 				<Dropdown
 					position="bottom left"
 					contentClassName="edit-post-post-schedule__dialog"
+					aria-labelledby="edit-post-post-schedule__heading"
 					renderToggle={ ( { onToggle, isOpen } ) => (
 						<Button
+							id="edit-post-post-schedule__toggle"
 							type="button"
 							className="edit-post-post-schedule__toggle"
 							onClick={ onToggle }
 							aria-expanded={ isOpen }
+							aria-live="polite"
+							aria-label={ __( 'Date and time' ) }
```

Comment:
Not sure about this, as using a label that's completely different from the displayed text won't work for some users, for example speech recognition software users. There's also #470 about the visibility and date buttons.

Reply:
So it's better to introduce a label tag instead?